include/crm/msg_xml_compat.h

```diff
@@ -109,6 +109,9 @@ extern "C" {
 //! \deprecated Use name member directly
 #define TYPE(x) (((x) == NULL)? NULL : (const char *) ((x)->name))
 
+//! \deprecated Do not use
+#define XML_CONFIG_ATTR_RECHECK "cluster-recheck-interval"
```

Comment:
All of the new symbols are internal and can't be included in the compat public header.

It's not unreasonable to make them public. However, there's probably not as much of a public use case for these as there is for, say, the CIB tags names, and we've been largely defaulting to private. Also the existing `PCMK__OPT_*` constants were private.

Reply:
**On the other hand**, we've been making `PCMK_META_*` constants public so far. There's about as much public use case for accessing properties as there is for accessing meta-attributes. Maybe we should consider private `PCMK__META` instead.

Reply:
Ah... we were already supposed to be doing that (see cc8ed479), but fdb2ead3 pulled us back to public lol.